main.go

```diff
@@ -203,7 +203,7 @@ func main() {
 		cli.StringFlag{
 			Name:   "commit.author.name",
 			Usage:  "git author name",
-			EnvVar: "DRONE_COMMIT_AUTHOR",
+			EnvVar: "DRONE_COMMIT_AUTHOR_NAME",
```

Comment:
`DRONE_COMMIT_AUTHOR` may not be mapped for your Git host, but it's definitely mapped for some. I think if you change this to `DRONE_COMMIT_AUTHOR,DRONE_COMMIT_AUTHOR_NAME`, it will pick the first one that's set.

Reply:
I've changed the mapping as requested and verified the plugin still maps the author on my Drone instance.